util/src/main/scala/com/scalableminds/util/io/ZipIO.scala

```diff
@@ -100,11 +100,24 @@ object ZipIO {
 
     def isFileHidden(e: ZipEntry): Boolean = new File(e.getName).isHidden || e.getName.startsWith("__MACOSX")
 
+    def stripPathFrom(path: Path, string: String): Option[String] = {
+      if(path.toString.contains(string))
```

Comment:
Could this directly operate on the path?

Reply:
Could it happen that string is just a substring of a directory? That looks dangerous to me then.

Reply:
You're right I'm looking into it